metrics-service/src/test/java/org/zowe/apiml/metrics/functional/MetricsFunctionalTest.java

```diff
@@ -28,12 +28,8 @@
     protected String hostname;
 
     @BeforeEach
-    void setUp() throws Exception {
+    void setUp() {
         RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
     }
 
-    protected String getDiscoveryUriWithPath(String path) {
-        return String.format("https://%s:%d", hostname, port) + path;
-    }
```

Comment:
Let's keep this function for now, it likely will be of use later when tests are written.